bitrise.yml

```diff
@@ -4,11 +4,12 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 app:
   envs:
   - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-artifacts.git
+  - BITRISE_AAB_PATH: ./app-bitrise-signed.aab
   - ORIG_BITRISE_SOURCE_DIR: $BITRISE_SOURCE_DIR
 
 workflows:
-  test:
-    title: Test
+  ci:
```

Comment:
I have seen that in other steps this workflow is rather called `ci`, so I have renamed it.